src/pages/library/dataBrowser-utils.js

```diff
@@ -71,6 +71,7 @@ const extractTags = snapshot => {
 }
 
 export const useDataCatalog = () => {
+  const skip = !isDataBrowserVisible()
```

Comment:
Var name suggestion?
```suggestion
  const skipMetadata = !isDataBrowserVisible()
```